WordPress/Classes/ViewRelated/NUX/SignupEmailViewController.swift

```diff
@@ -0,0 +1,221 @@
+import UIKit
+
+class SignupEmailViewController: NUXAbstractViewController, SigninKeyboardResponder {
```

Comment:
I think this would benefit from subclassing `LoginViewController` instead of `NUXAbstractViewController`. If you can forgive the login-specific name, it would allow you to remove some of the `@IBOutlet`s, give you a `displayError` implementation for free, and give the next button an `active` animation with an `enableSubmit` override. I'm feeling like `LoginViewController` will become `NUXViewController`.

Let me know what you think.